packages/shared/lib/src/sdk/args.rs

```diff
@@ -761,6 +765,8 @@ pub fn ibc_transfer_tx_args(
         channel_id,
         timeout_height,
         timeout_sec_offset,
+        // TODO: false for now
+        disposable_signing_key: false,
```

Comment:
What do you think about mapping this to the types/args before we release? I could do a follow-up PR that adds any potential args we may want soon to our types package, that way we wouldn't have to worry about any breaking APIs after we publish. Unless there's not a use-case we'll need for this. I can look into that!

Reply:
I think we can add it when we start testing transfers